src/protocol/tcp/src/main/java/org/apache/jmeter/protocol/tcp/sampler/TCPSampler.java

```diff
@@ -504,7 +504,7 @@ private void closeSocket(String socketKey) {
             try {
                 con.close();
             } catch (IOException e) {
-                log.warn("Error closing socket {}", e); //$NON-NLS-1$
+                log.warn("Error closing socket", e); //$NON-NLS-1$
```

Comment:
Again, I think it is better to keep the short (string version) of the message here.